rules-reviewed/eap6/jboss-eap5/jboss-eap5-java.windup.xml

```diff
@@ -52,6 +52,7 @@
                       ]]>
                     </message>
                     <link title="JBoss EAP 6 - Messaging Configuration" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html/Administration_and_Configuration_Guide/sect-Configuration1.html#Configure_the_JMS_Server1"/>
+                    <link title="JBoss EAP 7 - Messaging Configuration" href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/7.0/configuring-messaging/chapter-3-basic-configuration"/>
```

Comment:
This is not ruleset for EAP 7 so including link for EAP7 is not correct IMO see https://github.com/windup/windup-rulesets/blob/master/rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml#L52